cpp/ql/src/semmle/code/cpp/ir/implementation/unaliased_ssa/internal/AliasConfiguration.qll

```diff
@@ -14,3 +14,11 @@ class Allocation extends IRAutomaticVariable {
     none()
   }
 }
+
+class StageEscapeConfiguration extends string {
+  StageEscapeConfiguration() {
+    this = "StageEscapeConfiguration (unaliased_ssa)"
+  }
+
+  predicate useSoundEscapeAnalysis() { any() }
+}
```

Comment:
(copied from https://github.com/github/codeql/pull/5509)

Is there are need to wrap `useSoundEscapeAnalysis` in a class? How about just

```suggestion
predicate useSoundEscapeAnalysis() { any() }
```

Consider renaming the predicate to `phaseNeedsSoundEscapeAnalysis` to reflect that this predicate does not alone determine whether sound escape analysis is used.

I think we use the term _phase_ rather than _stage_ since a stage means something specific in QL: a set of `cached` predicates and their dependencies.